lib/unwrappr/researchers/github_comparison.rb

```diff
@@ -14,14 +14,14 @@ def initialize(client)
 
       def research(gem_change, gem_change_info)
         repo = gem_change_info[:github_repo]
-        return gem_change_info if repo.nil?
+        return gem_change_info if repo.nil? ||
```

Comment:
long if's make me pause to parse.
Wonder if we can name this a bit better?
`return gem_change_info unless gem_change_information_available?` (or something similar?)

Reply:
I feel it's important to inform the reader what the conditions are here, rather than hiding behind an opaque method name. What about something like this:

```ruby
return gem_change_info if github_repo_not_identified?(gem_change_info) ||
                          gem_added?(gem_change) ||
                          gem_removed?(gem_change)
```

Reply:
I'll go with this last one for now. Feel free to discuss further. 😃

Reply:
yeah I'm not super fussed with the 3 ||'s just when I read it it should reveal the intent a little clearer. both you're suggestions read better to me. Thanks

Reply:
Cheers. It was a good call.